core/gui/src/app/hub/component/workflow/detail/hub-workflow-detail.component.ts

```diff
@@ -161,7 +161,9 @@ export class HubWorkflowDetailComponent implements AfterViewInit, OnDestroy, OnI
   }
 
   goBack(): void {
-    this.location.back();
+    this.router.navigateByUrl("/dashboard/hub/workflow/result").catch(error => {
```

Comment:
this address is hard coded and could have a two-copy issue. Please make the `/dashboard/hub/workflow/result` a constant variable in the navigation-related file (possibly app-routing module), export it, and reference it here to avoid the two-copy issue.

also, please remove the console log. if you need to alert the user, use the notification service to send out an error message that is rendered to user.